app/code/core/Mage/Core/Model/Design/Package.php

```diff
@@ -720,7 +728,15 @@ public function getMergedCssUrl($files)
         }
 
         // merge into target file
-        $targetFilename = md5(implode(',', $files) . "|{$hostname}|{$port}") . '.css';
+        $newestTimestamp = '';
```

Comment:
```suggestion
        $newestTimestamp = 0;
```

since later we're comparing the timestamp values I think it's better if they're of the same type